recipe/meta.yaml

```
@@ -163,12 +163,12 @@ requirements:
     # http://doc.qt.io/Qt-5/qdoc-guide-clang.html
     # Only libclang is needed, but needs the full package for detection.
     - llvmdev
+    - clangdev
     # Ensure that build and host have the same clang version on osx
     # otherwise qt6 may be out of sync compared to the pinned compiler
     # version
-    - clangdev {{ cxx_compiler_version }}  # [osx]
-    - clangdev                             # [not osx]
-    - libclang
+    - libclang {{ cxx_compiler_version }}  # [osx]
+    - libclang                             # [not osx]
```

If we're pinning libclang, this should be
```suggestion
    - libclang13
```

Though to be fair, that's just the "should be" version, and wouldn't solve the [problem](https://github.com/conda-forge/pyside2-feedstock/issues/159#issuecomment-1356403824) that you ran into in for pyside. That error is still mysterious to me, because those builds _do_ exists.

---

So I've played around with this in https://github.com/conda-forge/pyside2-feedstock/pull/165, and I think your solution is good enough now and should be merged.

Medium term, we should probably add `libclang` to the global pinning and do something similar to numpy - build for the oldest abi-compatible version (in this case libclang =13), and/or change the relationship between `clangdev` and `libclang` so that newer but abi-compatible libclang can be installed alongside.